src/util-provisioning-profiles.ts

```diff
@@ -83,12 +85,12 @@ export async function getProvisioningProfile(filePath: string, keychain: string
  */
 export async function findProvisioningProfiles(opts: ValidatedSignOptions) {
   const cwd = process.cwd();
-  const children = await fs.readdir(cwd);
+  const children = await util.promisify(fs.readdir)(cwd);
```

Comment:
Oh, should we just use `fs.promises.readdir` here?

Reply:
`graceful-fs` unfortunately comes with the side effect of needing to `util.promisify` because it only patches callback-based functions. Ref: https://github.com/isaacs/node-graceful-fs/issues/160